src/Microsoft.AspNetCore.Mvc.RazorPages/Infrastructure/DefaultPageActivator.cs

```diff
@@ -12,10 +13,10 @@ namespace Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure
     /// </summary>
     public class DefaultPageActivator : IPageActivatorProvider
```

Comment:
If you feel up to it, can you change the type name to `DefaultPageActivatorProvider`?